src/MEDS_transforms/extract/shard_events.py

```diff
@@ -227,7 +227,14 @@ def retrieve_columns(event_conversion_cfg: DictConfig) -> dict[str, list[str]]:
     prefix_to_columns = {}
 
     default_subject_id_col = event_conversion_cfg.pop("subject_id_col", subject_id_field)
+    tables_to_ignore = event_conversion_cfg.pop("tables_to_ignore", None)
+    if tables_to_ignore:
+        logger.warning(f"Ignoring tables: {tables_to_ignore}")
```

Comment:
I think this shouldn't be a warning necessarily as these were requested to be ignored.